tools/SourceKit/tools/sourcekitd/lib/Service/Requests.cpp

```diff
@@ -1682,6 +1688,31 @@ handleRequestRelatedIdents(const RequestDict &Req,
   });
 }
 
+static void
+handleRequestInactiveRanges(const RequestDict &Req,
+                            SourceKitCancellationToken CancellationToken,
+                            ResponseReceiver Rec) {
+  if (checkVFSNotSupported(Req, Rec))
+    return;
+
+  handleSemanticRequest(Req, Rec, [Req, CancellationToken, Rec]() {
+    auto SourceFile = getSourceFileNameForRequestOrEmitError(Req, Rec);
+    if (!SourceFile)
+      return;
+    SmallVector<const char *, 8> Args;
```

Comment:
We no longer need to specify a size for the `SmallVector`, so the following should be sufficient and `SmallVector` will pick a default size that it finds suitable.

```suggestion
    SmallVector<const char *> Args;
```

Reply:
Ah, nice. All other requests in this file still have that, though. But I'll just apply your suggestion for this new code. Not attempting to modernize all other requests while I'm here.